content/en/docs/releases/next-release-preview/index.md

```diff
@@ -46,15 +46,20 @@ that this SpEL should be evaluated by a different tool. No exceptions are thrown
 ${#toJson(#doNotEval(fileMap))}
 ```
 
-Use a feature flag to enable.
+This feature introduces a new SpEL `doNotEval` method that includes the received JSON object with the NotEvaluableExpression class. 
+The toJson method (and others in the future) will not evaluate expressions and will not throw exceptions for instances of the NotEvaluableExpression class.
+- The feature flag introduced in Kork (1.28/29 releases) to use `doNotEval` SPeL expression helper will be enabled by default.
+- This feature is disabled only when the flag is set explicitly to false.
+
+  ```yaml
+    # orca-local.yml
+  
+  expression:
+    do-not-eval-spel:
+      enabled: false
+  ```
+See the changes [here](https://github.com/spinnaker/kork/pull/1028)
```

Comment:
Minor but I'd link to:
https://github.com/spinnaker/kork/pull/978
as that's the root of what's going/explains the details.  The other is the feature flag PR vs. what that feature flag does.

Reply:
That's right, the link has been modified accordingly.